script.js

```diff
@@ -1,4 +1,115 @@
+// global variables
+var userName = ``;
+var numPlayerWinCounter = 0;
+var numComputerWinCounter = 0;
+var numDrawsCounter = 0;
+
+// game modes
+
+var gameMode1 = `user to log in`;
+var gameMode2 = `user to throw`;
+var gameMode3 = `user to play`;
+gameMode = gameMode1;
+
+// generate random number
+var generateRandomNo = function () {
+  var randomNo = Math.ceil(Math.random() * 3);
+  return randomNo;
+};
+
+// define outcome
+var numberDefinition = function (generatedinteger) {
+  var computerAction = `Default`;
+  var computerPick = generateRandomNo();
+
+  if (computerPick == 1) {
+    computerAction = `scissors`;
+  }
+
+  if (computerPick == 2) {
+    computerAction = `paper`;
+  }
+
+  if (computerPick == 3) {
+    computerAction = `stone`;
+  }
+
+  return computerAction;
+};
```

Comment:
Goo work with this function! Would be better to name this function more accurately. 

e.g. `generateSPS `